tools/osx-pkg/README.md

```diff
@@ -0,0 +1,46 @@
+## osx-pkg
+
+### Build
+
+Prerequisites:
+
+* Packages: http://s.sudre.free.fr/Software/Packages/about.html
+
+In the root folder:
+
+```bash
+make pkg
+```
+
+### Localization
+
+There are two files that can be localized in the OS X installer: the 
+introduction, and the summary.
+
+1. Make sure you've installed Packages: 
+http://s.sudre.free.fr/Software/Packages/about.html
```

Comment:
indent all of these so the second and subsequent lines line up with the start of the first, e.g.

```
1. Foo bar ...
    whiz bang
```